### PR TITLE
change colour of error from red to error-background variable value

### PR DESCRIPTION
**Pull request by imran-salim**
https://dev.azure.com/S72/SHIFT72/_workitems/edit/4972

### CODE REVIEW

File: site/styles/_forms.scss [hunk 1]
```diff
@@ -70,6 +70,9 @@ s72-activatedevice-form,
     top: -5px;
   }
 }
+.s72-form-errors {
+  color: $error-color;
+}
```
Comment (stajs) [inline site/styles/_forms.scss:75]:
```suggestion

.s72-form-errors {
  color: $error-color;
}

```

**Review by stajs — APPROVED** [review]
Need a line break before/after the class, otherwise all good.

BTW, I don't know how you managed to mess the link up but I fixed it for you ;)

image

image

**Comment by stajs** [comment]
@PeculiarGoat Can you have a look if we can get rid of these failing checks?

**Comment by PeculiarGoat** [comment]
PLEASE DON'T MERGE THIS... it's supposed to be a Relish fix... should probably go in core-team's backlog.

**Comment by stajs** [comment]
@PeculiarGoat you should "request changes" on this PR (or possibly close with comment) to prevent merging if you are against.

But where is the change? Isn't it a style change? So core template?

**Comment by imran-salim** [comment]
Okay, I'll delete this branch RIP

**Comment by imran-salim** [comment]
This bug requires a Relish fix rather than a style fix.

**Comment by stajs** [comment]
What's the relish fix?

**Comment by stajs** [comment]
What's the relish fix?

**Comment by PeculiarGoat** [comment]
@stajs

This is what the errors are supposed to look like:
```
<div class="s72-form-errors">
  <span class="s72-error-message s72-error-message-required">Please enter a password.</span>
</div>
```

This is what that particular error looks like:
```
<div class="s72-form-errors">That password is incorrect.</div>
```

Maybe the second example can be styled to look like the first?  But IMO this should be fixed in the Relish component.

**Comment by imran-salim** [comment]
I couldn't locate anything in the repo with the tags <div class="s72-form-errors"> or <span class="s72-error-message s72-error-message-required">

**Comment by imran-salim** [comment]
I couldn't locate anything in the repo with the tags `<div class="s72-form-errors">` or `<span class="s72-error-message s72-error-message-required">`

**Comment by stajs** [comment]
@imran-salim It's not in this repo.

**Pull request closed without merging**